src/Storages/System/StorageSystemDDLWorkerQueue.cpp

```diff
@@ -62,7 +62,7 @@ ColumnsDescription StorageSystemDDLWorkerQueue::getColumnsDescription()
         {"entry_version",       std::make_shared<DataTypeNullable>(std::make_shared<DataTypeUInt8>()), "Version of the entry."},
         {"initiator_host",      std::make_shared<DataTypeNullable>(std::make_shared<DataTypeString>()), "Host that initiated the DDL operation."},
         {"initiator_port",      std::make_shared<DataTypeNullable>(std::make_shared<DataTypeUInt16>()), "Port used by the initiator."},
-        {"cluster",             std::make_shared<DataTypeString>(), "Cluster name."},
+        {"cluster",             std::make_shared<DataTypeNullable>(std::make_shared<DataTypeString>()), "Cluster name if determined."},
```

Comment:
Actually I guess we can simply use plain `String` (w/o `Nullable`), and avoid type change, since we should always have some cluster here.
Sorry for missing this the first time, let's remove `Nullable`

Reply:
@azat , to me both ways are equal, don't see meaningful pros and cons.
The original string is "UNKNOWN". Is it Ok?

Reply:
UNKNOWN can be a cluster name, while cluster cannot be empty, so let's use empty string